src/_data/links.json

```diff
@@ -1,8 +1,9 @@
 {
     "github": "https://github.com/eslint/eslint",
-    "twitter": "https://twitter.com/geteslint",
     "chat": "https://eslint.org/chat",
+    "bluesky": "https://bsky.app/profile/eslint.org",
     "mastodon": "https://fosstodon.org/@eslint",
+    "twitter": "https://twitter.com/geteslint",
```

Comment:
I couldn't resist putting the three social media networks together, in alphabetical order...